trajopt_optimizers/trajopt_optimizers_test_suite/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+cmake_minimum_required(VERSION 3.5.0)
+project(trajopt_optimizers_test_suite VERSION 0.1.0 LANGUAGES CXX)
+
+find_package(console_bridge REQUIRED)
+find_package(Eigen3 REQUIRED)
+find_package(ifopt REQUIRED)
+find_package(OsqpEigen REQUIRED)
+find_package(trajopt_ifopt REQUIRED)
+find_package(trajopt_utils REQUIRED)
+find_package(trajopt_sqp REQUIRED)
+
+###########
+## Build ##
+###########
+
+add_library(${PROJECT_NAME} SHARED
+    src/run_tests.cpp
+)
+target_include_directories(${PROJECT_NAME}
+  PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include>
+)
+target_link_libraries(${PROJECT_NAME} PUBLIC
+    ifopt::ifopt_core
+    console_bridge
+    OsqpEigen::OsqpEigen
```

Comment:
I would think that we don't need osqp_eigen. I also think you need trajopt_ifopt and can drop ifopt_core since trajopt_ifopt should have that already